client/graph.go

```diff
@@ -41,5 +41,6 @@ type SolveStatus struct {
 }
 
 type SolveResponse struct {
-	ExporterResponse map[string]string
+	ExporterResponse      map[string]string
+	CacheExporterResponse map[string]string
```

Comment:
Is this needed as `ExporterResponse` is already generic. Maybe we can just properly namespace the cache keys.

Reply:
added `cache.manifest` key